internal/cmd/set.go

```diff
@@ -126,7 +126,6 @@ func SetCommand() cli.Command {
 	}
 }
 
-func makeProxyURL(port int) string {
-	p := fmt.Sprintf("http://localhost:%v", port)
-	return p
+func setProxyURL(port int) string {
```

Comment:
so you changed it back to `set...`? I expect a `set` function/method to have side-effects, which is why I was suggesting renaming it. but up to you.

Reply:
Refactored to makeProxyURL